src/Entity/Package.php

```diff
@@ -539,6 +542,14 @@ public function getBrowsableRepository(): string
     public function addVersion(Version $version): void
     {
         $this->versions[] = $version;
+
+        $newVersion = strtolower($version->getNormalizedVersion());
+
+        if (null === $this->snapshotedLatestVersion
+            ||
+            version_compare($newVersion, $this->snapshotedLatestVersion, '>=')) {
+            $this->snapshotedLatestVersion = $newVersion . '&' . (new \DateTime('now'))->format('Y-m-d');
```

Comment:
the date of adding the data in the entity is not the date of that version (it depends when the sync happens)

Reply:
indeed! we may leverage `Version::releasedAt` here?